### PR TITLE
Add flex class Readme (for tomorrow)

### PR DESCRIPTION
**Pull request by EvanDorsky**

### CODE REVIEW

File: classes/class05/README.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+In the real world (and in this class), it's not good practice to commit directly to the `master` branch. If a real project is under version control, committing a bug to `master` could break live code. The `master` branch is generally reserved for code that has been tested and shown to work in a branch.
+
+1. Create a new branch on your olinjs repository.
+
+```bash
+$ git checkout -b branchname
+```
+
+1. Commit something on the branch.
+1. Pull request that branch on GitHub.
```
Comment (EvanSimpson) [inline classes/class05/README.md]: pull request where? origin or their forks?

File: classes/class05/README.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+In the real world (and in this class), it's not good practice to commit directly to the `master` branch. If a real project is under version control, committing a bug to `master` could break live code. The `master` branch is generally reserved for code that has been tested and shown to work in a branch.
+
+1. Create a new branch on your olinjs repository.
+
+```bash
+$ git checkout -b branchname
+```
+
+1. Commit something on the branch.
```
Comment (EvanSimpson) [inline classes/class05/README.md]: numbering starts over from 1 here because of the code break.

**Comment by EvanSimpson** [comment]
Those 2 comments, but :+1:

File: classes/class05/README.md (hunk 1 above)
Comment (EvanDorsky) [inline classes/class05/README.md]: Good point, we'll also explain this in class.

File: classes/class05/README.md (hunk 2 above)
Comment (EvanDorsky) [inline classes/class05/README.md]: Got it, thanks.

**Comment by benkahle** [comment]
Add a link from the homepage please

**Comment by benkahle** [comment]
:+1:

**Pull request merged**